JS.md

```diff
@@ -608,6 +619,30 @@ accessor function performs the following steps:
 If `this` is not a `WebAssembly.Memory`, a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror)
 is thrown. Otherwise return `M.[[BufferObject]]`.
 
+### `WebAssembly.Memory.prototype.length`
```

Comment:
Sounds good. That's what I had but @rossberg-chromium disliked it. I'll put it back!